specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/examples/getStorageSasDefinition.json

```diff
@@ -0,0 +1,25 @@
+{
+   "parameters":{
+      "vaultBaseUrl":"https://storage-sdk-test.vault-int.azure-int.net/",
+      "storage-account-name":"getsas01",
+      "sas-definition-name":"getStrgSasDef01",
+      "api-version":"2016-10-01"
+   },
+   "responses":{
+      "200":{
+         "headers":{
+
+         },
+         "body":{
+            "id":"https://storage-sdk-test.vault-int.azure-int.net/storage/getsas01/sas/getStrgSasDef01",
+            "sid":"https://storage-sdk-test.vault-int.azure-int.net/secrets/getsas01-getStrgSasDef01",
+            "template":"se=2018-02-01T00%3A00Z&spr=https&sv=2017-04-17&sr=b&sig=XXFNfuMCHYrBx0bhemJ7PWn0xGfImMXT6LfbXWvtRUk%3D",
```

Comment:
there are 2 new parameters: "sasType" and "validityPeriod", in all sas get, delete, update, set responses and set request